src/App.js

```diff
@@ -2,18 +2,94 @@ import React from 'react';
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-}
+import { List } from './components/List';
+import { Form } from './components/Form';
+
+const todoList = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
+
+export class App extends React.Component {
+  state = {
+    newTodo: '',
+    chosedUser: '',
+    userTodos: todoList,
```

Comment:
зачем добавлять слово юзер? делай названия простыми максимально
```suggestion
    todos: todoList,
```